packages/react-components/src/templates/DiscoverGuides.tsx

```diff
@@ -0,0 +1,106 @@
+import { css } from '@emotion/react';
+
+import { HelpSection } from '../organisms';
+import { perRem } from '../pixels';
+import { Headline2, Paragraph } from '../atoms';
+import { Accordion } from '../molecules';
+import {
+  giftIcon,
+  dashboardIcon,
+  budgetIcon,
+  reportIcon,
+  shareIcon,
+  moneyIcon,
+  toolsIcon,
+} from '../icons';
+import policyIcon from '../icons/policy';
+
+const styles = css({
+  display: 'grid',
+  gridRowGap: `${36 / perRem}em`,
+  paddingBottom: `${24 / perRem}em`,
+});
+
+const DiscoverGuides: React.FC = () => (
+  <div css={styles}>
+    <div>
+      <Headline2 styleAsHeading={3}>Guides</Headline2>
+      <Paragraph accent="lead">
+        Explore our guides, templates and forms to help you with your daily
+        activities.
+      </Paragraph>
+    </div>
+    <Accordion
+      items={[
+        {
+          icon: giftIcon,
+          title: 'Grant Welcome Packet',
```

Comment:
Not for this ticket, but this looks like something that should be in the CMS and not hard codded in the code

Reply:
Yeah, the initial plan is to have it static then we move it later in the CMS